1001-2000/LIT1011Abusak.xml

```diff
@@ -68,10 +80,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <bibl>
                      <ptr target="bm:Neugebauer1988Abushaker"/>
                   </bibl>
+                  <bibl>
+                    <ptr target="bm:Uhlig2003Abushaker"/>
+                  </bibl>
+                  <bibl>
+                    <ptr target="bm:Ricci1967Enbaqom"/>
+                  </bibl>
                </listBibl>
             </listBibl>
          </div>
          <div type="edition">
```

Comment:
I know it was not there before, but maybe you could add an explicit statement on the source of this structure? I think that's always good.